### ISSUE TITLE
Warning for people watching github: email flood due to ticket conversion

### ISSUE THREAD

**Issue by auriamg** [supertuxkart/stk-code#55]
Hi,

this is not actually a ticket, but we are using this to reach people watching the SuperTuxKart project on github,njmm

we will be converting all our trac tickets from sourceforge to github.
Since github will send out an email for every new ticket and every
comment, we expect 1000 or more emails to be sent to anyone watching
supertuxkart on github.

We would recommend to stop watching supertuxkart on github for a day or
two to avoid this flood. We will post on our mailing list once the migration is complete
